### PR TITLE
Update Proposal26.t.sol

### PR DESCRIPTION
**Pull request by rossgalloway**
- Added additional descriptions with comments.
- indented the whitelist array inputs for clarity

### CODE REVIEW

**Comment by rossgalloway** [comment]
Not sure if the indenting something like the whitelist tokens is good or bad code practice, but it makes it much easier to understand that there are different inputs to the array for the casual viewer.

File: contracts/2022/05/Proposal26.t.sol [hunk 1]
```diff
@@ -59,45 +59,48 @@ contract Proposal26test is YAMTest {
...
+            amounts[4] = type(uint256).max;
+            tokens[4] = address(SCJAN6);
+
+            whos[5] = address(proposal);
+            amounts[5] = type(uint256).max;
+            tokens[5] = address(SCDEC2);
+
+            whos[6] = address(proposal);
+            amounts[6] = type(uint256).max;
+            tokens[6] = address(SCNOV3);
```
Comment (ethedev) [inline contracts/2022/05/Proposal26.t.sol]:
```suggestion
        address[] memory whos = new address[](7);
        uint256[] memory amounts = new uint256[](7);
        address[] memory tokens = new address[](7);

        //whitelist USDC
        whos[0] = address(proposal);
        amounts[0] = type(uint256).max;
        tokens[0] = address(USDC);

        //whitelist YAM
        whos[1] = address(proposal);
        amounts[1] = type(uint256).max;
        tokens[1] = address(YAM);

        //whitelist UMA synths and success tokens
        whos[2] = address(proposal);
        amounts[2] = type(uint256).max;
        tokens[2] = address(UGASDEC21);

        whos[3] = address(proposal);
        amounts[3] = type(uint256).max;
        tokens[3] = address(UPUNKSDEC21);

        whos[4] = address(proposal);
        amounts[4] = type(uint256).max;
        tokens[4] = address(SCJAN6);

        whos[5] = address(proposal);
        amounts[5] = type(uint256).max;
        tokens[5] = address(SCDEC2);

        whos[6] = address(proposal);
        amounts[6] = type(uint256).max;
        tokens[6] = address(SCNOV3);
```

File: contracts/2022/05/Proposal26.t.sol [hunk 2]
```diff
@@ -59,45 +59,48 @@ contract Proposal26test is YAMTest {
...
         calldatas[0] = abi.encode(whos, amounts, tokens);
 
+        // claim sushi rewards from incentivizer
```
Comment (ethedev) [inline contracts/2022/05/Proposal26.t.sol]:
```suggestion
        // Claim sushi rewards from incentivizer
```

File: contracts/2022/05/Proposal26.t.sol [hunk 3]
```diff
@@ -108,6 +111,7 @@ contract Proposal26test is YAMTest {
         tokensOne[0] = address(UMA);
         calldatas[2] = abi.encode(whosOne, amountsOne, tokensOne);
 
+        //withdraw 1 UMA from twoKeyContract
```
Comment (ethedev) [inline contracts/2022/05/Proposal26.t.sol]:
```suggestion
        // Withdraw 1 uma from twoKeyContract
```

**Comment by ethedev** [comment]
> Not sure if the indenting something like the whitelist tokens is good or bad code practice, but it makes it much easier to understand that there are different inputs to the array for the casual viewer.

Yes, bad code practice

**Pull request merged**